lgmys/kibana
# [TIP] Integrate flyout component with indicator table

## Summary

Integrate the flyout component with the indicate page and table.

Add the actions column on the left side of the table using the available `leadingControlColumns` feature of the `EuiDataGrid` component.

The `OpenIndicatorFlyoutButton` is added to Storybook

https://user-images.githubusercontent.com/17276605/178374826-987e3dda-7674-442d-9b16-c6cb853acbc8.mov

Surprisingly there is a weird bug when loading the component from the app, where the title is black even in dark mode. The bug doesn't happen in Storybook though...


https://user-images.githubusercontent.com/17276605/178374890-37043510-89e3-4e3a-8d9d-ce8c5b74fab2.mov

### How to test

- make sure you have data flowing (for example using Filebeat), then click on the expand icon at the beginning of the row

```
unit test:
npm run test:jest --config ./x-pack/plugins/threat_intelligence

storybook:
node scripts/storybook threat_intelligence
```

https://github.com/elastic/security-team/issues/4245
https://github.com/elastic/security-team/issues/4244
